_layouts/department.html

```diff
@@ -14,6 +14,13 @@
 {% endif %}
 
 
+{% if page_data.government.slug == 'south-africa' %}
+  {% assign source_type = 'Estimates of National Expenditure' %}
+{% else %}
+  {% assign source_type = 'Estimates of Provincial Revenue and Expenditure' %}
```

Comment:
Please change this to "Estimates of Provincial Expenditure"

Reply:
Will do. Should I change it everywhere on the page? I borrowed the above from other mentions of it on the page.

Reply:
I think some of it is also pulled from what JD provides me as well.

Reply:
Good idea. Please open a separate pull request for that, let's get this one out.